Dockerfile

```diff
@@ -7,4 +7,6 @@ ADD . /go/src/github.com/redhatanalytics/oshinko-rest
 WORKDIR /go/src/github.com/redhatanalytics/oshinko-rest
 RUN make install
 
+RUN chmod a+rwX -R .
```

Comment:
this totally works, do you think we should come back at some point and restrict the execute perms to only hit the files we care about?